codespell_lib/data/dictionary_rare.txt

```diff
@@ -91,6 +91,7 @@ preform->perform
 preformed->performed
 preforms->performs
 pres->press
+prioritary->priority, prioritary,
```

Comment:
Can you drop the correction to itself please.

Reply:
3ce113e